### ISSUE TITLE
Distance does not work on mobile

### ISSUE THREAD

**Issue by LeoIannacone** [clauderic/react-sortable-hoc#176]
It looks like that it only works if `distance` is undefined in mobile.

**Comment by corneldm** [comment]
I'm new to this library and can confirm some issues with mobile. `distance` doesn't cooperate, nor does using `SortableHandle` in my experience. I'm looking around the source to see if I can help with a PR.

**Comment by corneldm** [comment]
My issue with `SortableHandle` is resolved (I was accidentally building multiple SortableHandle components). `distance` still seems to be an issue, however.

**Comment by clauderic** [comment]
Hey @LeoIannacone, are you using the `distance` prop in conjunction with the `pressDelay` prop? If so, you can only use one or the other, not both at the same time

**Comment by LeoIannacone** [comment]
This is actually my call:
```
    const distance = this.context.mobileDetect.phone() ? undefined : 5
    return (
      <ServiceSelectorSorterListSortable
        myData={data}
        onSortEnd={this._onSortEnd.bind(this)}
        distance={distance}
        helperClass='ServiceSelectorSorterItem--sorting'
        />
    )
```

**Comment by alexdemartos** [comment]
Same issues here: distance not working on my Nexus 5, nor Desktop Chrome dev tools mobile-like template. Not using pressDelay.

PD: pressDelay is working fine both on desktop & mobile browsers.

**Comment by viniciusvgv** [comment]
The same happens to me, `distance` doesn't work on touch screens (I have the `0.6.3` version and tested using Chrome's mobile device inspector). `pressDelay` works just fine but with the layout I have `distance` feels much better.

**Comment by clauderic** [comment]
Fixed by #378
